docs/tap_hold.md

```diff
@@ -425,6 +425,243 @@ uint16_t get_quick_tap_term(uint16_t keycode, keyrecord_t *record) {
+
+- `SFT_T(KC_A)` Down
+- `KC_C` Down
+- `KC_C` Up
+- `SFT_T(KC_A)` Up
+
+```
+                         TAPPING_TERM   
+  +---------------------------|--------+
+  | +----------------------+  |        |
+  | | SFT_T(KC_A)          |  |        |
+  | +----------------------+  |        |
+  |    +--------------+       |        |
+  |    | KC_C         |       |        |
+  |    +--------------+       |        |
+  +---------------------------|--------+
+```
+
+If the two keys are on the same hand, then this will produce `ac` with 
+`SFT_T(KC_A)` settled as tapped the moment that `KC_C` is pressed. 
+
+If the two keys are on opposite hands and the `HOLD_ON_OTHER_KEY_PRESS`
+option enabled, this will produce `C` with `SFT_T(KC_A)` settled as held the
+moment that `KC_C` is pressed.
+
+Or if the two keys are on opposite hands and the `PERMISSIVE_HOLD` option is
+enabled, this will produce `C` with `SFT_T(KC_A)` settled as held the
+moment that `KC_C` is released.
+
+### Chordal Hold Handedness
+
+Determining whether keys are on the same or opposite hands involves defining the
+"handedness" of each key position. By default, if nothing is specified,
+handedness is guessed based on keyboard matrix dimensions. If this is
+inaccurate, handedness may be specified in several ways.
+
+The easiest way to specify handedness is by `chordal_hold_layout`. Define in
+config.h:
+
+```c
+#define CHORDAL_HOLD_LAYOUT
+```
+
+Then in keymap.c, define `chordal_hold_layout` in the following form:
+
+```c
+const char chordal_hold_layout[MATRIX_ROWS][MATRIX_COLS] PROGMEM =
+    LAYOUT(
+        'L', 'L', 'L', 'L', 'L', 'L',  'R', 'R', 'R', 'R', 'R', 'R', 
+        'L', 'L', 'L', 'L', 'L', 'L',  'R', 'R', 'R', 'R', 'R', 'R', 
+        'L', 'L', 'L', 'L', 'L', 'L',  'R', 'R', 'R', 'R', 'R', 'R', 
+                       'L', 'L', 'L',  'R', 'R', 'R'
+    );
+```
+
+Use the same `LAYOUT` macro as used to define your keymap layers. Each entry is
+a character, either `'L'` for left, `'R'` for right, or `'*'` to exempt keys
+from the "opposite hands rule." When a key has `'*'` handedness, pressing it
+with either hand results in a hold. This could be used perhaps on thumb keys or
+other places where you want to allow same-hand chords.
+
+Alternatively, handedness may be defined functionally with
+`chordal_hold_handedness_user()`. For example, in keymap.c define:
+
+```c
+char chordal_hold_handedness_user(keypos_t key) {
+    if (key.col == 0 || key.col == MATRIX_COLS - 1) {
+        return '*';  // Exempt the outer columns.
+    }
+    // On split keyboards, typically, the first half of the rows are on the
+    // left, and the other half are on the right.
+    return key.row < MATRIX_ROWS / 2 ? 'L' : 'R';
+}
+```
+
+Given the matrix position of a key, the function should return `'L'`, `'R'`, or
+`'*'`. Adapt the logic in this function according to the keyboard's matrix.
+Similarly at the keyboard level, `chordal_hold_handedness_kb()` may be defined
+to specify handedness.
+
```

Comment:
```suggestion

::: tip If you have both `chordal_hold_layout[MATRIX_ROWS][MATRIX_COLS]` and `chordal_hold_handedness_user(keypos_t key)` created for handedness, the latter takes precedence.

```

Reply:
Thank you for the suggestion. I have incorporated it into the documentation.

In a separate thread, I revised handedness configuration so that in descending order of precedence, handedness is defined by:

1. (user level) a strong definition of `chordal_hold_handedness()`
2. (user level) a strong definition of `chordal_hold_layout`
3. (keyboard level) a weak definition of `chordal_hold_layout`, which is auto generated  from keyboard.json. The handedness of a key is assumed based on its center x position relative to the median x value, or the keyboard maker may use an optional `"hand"` field to set it explicitly.